src/workerd/io/worker-interface.capnp

```diff
@@ -214,9 +214,10 @@ interface EventDispatcher @0xf20697475ec1752d {
   # the outcome and whether the run should be retried. This does not complete immediately.
 
 
-  runAlarm @4 (scheduledTime :Int64) -> (result :AlarmRun);
+  runAlarm @4 (scheduledTime :Int64, isRetry :Bool) -> (result :AlarmRun);
   # Runs a worker's alarm.
   # scheduledTime is a unix timestamp in milliseconds for when the alarm should be run
+  # isRetry indicates wether this runAlarm is a retry or not.
```

Comment:
```suggestion
  # isRetry indicates whether this runAlarm is a retry or not.
```

Reply:
Currently, each retry is counted as a different alarm invocation, and we have no way to differentiate them. We want to be able to distinguish retries from regular invocation.

Reply:
I was just trying to point out the spelling typo in the comment ("whether", not "wether")